dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/Director.java

```diff
@@ -507,21 +503,11 @@ public ServerPackageAsset resolveServerPackage(File archiveFile) throws InstallE
                 serverNames.add(sa.getServerName());
 
             }
-
```

Comment:
This part (line 510-524)  is the change I made. All the other ones are due to indentation changes.